src/test/java/lotto/domain/LottoTest.java

```diff
@@ -14,8 +15,9 @@ class LottoTest {
     @Test
     void 로또_생성을_확인한다() {
         //given
-        Lotto lotto = new Lotto(List.of(new LottoNumber(1), new LottoNumber(2), new LottoNumber(3), new LottoNumber(4),
-                new LottoNumber(5), new LottoNumber(7)));
+        Lotto lotto = new Lotto(
+                new HashSet<>(List.of(new LottoNumber(1), new LottoNumber(2), new LottoNumber(3), new LottoNumber(4),
+                        new LottoNumber(5), new LottoNumber(6))));
```

Comment:
```suggestion
        Lotto lotto = new Lotto(
            Set.of(
                new LottoNumber(1),
                new LottoNumber(2),
                new LottoNumber(3),
                new LottoNumber(4),
                new LottoNumber(5), 
                new LottoNumber(6)
            )
        );
```
코드가 정리되어 있으면 조금 더 읽기 편하지 않을까요?